Dockerfile

```
@@ -74,6 +75,7 @@ RUN	apt update && apt install -yqq \
 ADD ./scripts/livepeer-vmagent /usr/local/bin
 COPY --from=catalyst-build	/opt/bin/		/usr/local/bin/
 COPY --from=node-build		/app/go-tools/w3	/opt/local/lib/livepeer-w3
+COPY --from=ffmpeg-build	/usr/local/bin/*		/usr/local/bin/
```

@hjpotter92 I can't seem to get this to work, I've added the copy in multiple places but still not seeing the binaries when i run livepeer in a box locally. any idea how i should be doing this?

---

`/usr/local/bin` gets overridden by `make box-dev` and mounts your local `bin` directory instead - that's how binaries built outside the container get in. You could try this?
```suggestion
COPY --from=ffmpeg-build	/usr/local/bin/*		/usr/bin/
```
Buuuuuut that still won't work with inplace-deploy. We're either going to need to ship a static ffmpeg alongside the other binaries or statically link it like we're doing with go-livepeer.